### PR TITLE
Get the GetCTC privacy policy fully translated

### PR DESCRIPTION
**Pull request by embarnard**

### CODE REVIEW

File: config/locales/en.yml [hunk 1]
```diff
@@ -2302,146 +2302,107 @@ en:
...
+        - Characteristics of protected classifications, such as gender, race, and ethnicity
+        - Tax information, such as social security number or individual taxpayer identification number (ITIN)
+        - State-issued identification, such as driver’s license number
+        - Financial information, such as employment, income and income sources
+        - Banking details for direct deposit of refunds
+        - Details of dependents
+        - Household information and information about your spouse, if applicable
+        - Information from your computer or device, such as IP address, operating system, browser, date and time of visit, and click-stream data (the website or domain you came from, the pages you visit, and the items you click on during your session)
+        07_info_required_by_irs: We collect information as required by the IRS in Revenue Procedure “Rev RP-21-24.”
+        '08_how_we_collect': How we collect your information
```
Comment (embarnard) [inline config/locales/en.yml:2329]: Not sure why they want us to add quotes to this key name, but the normalize check fails unless these are here @alaroia-cfa

Comment (alaroia-cfa) [reply config/locales/en.yml:2329]: @embarnard Curious, and OK!

File: config/locales/es.yml [hunk 2]
```diff
@@ -2048,6 +2048,82 @@ es:
           verification_code_label: 'Introduzca el código de 6 números '
           verify: Verifique
     ctc_pages:
+      07_info_required_by_irs: Recopilamos información según lo dispuesto por el IRS en el Procedimiento de Ingresos "Rev RP-21-24".
```
Comment (alaroia-cfa) [inline config/locales/es.yml:2051]: The `07_info_required_by_irs` key here seems to be nested directly under `ctc_pages`. In the English, it's nested under `privacy_policy`. I think the Spanish keys here need to move.

**Review by alaroia-cfa — CHANGES_REQUESTED** [review]
Getting close -- I think some Spanish keys need to move. Thanks for taking this on.

**Comment by alaroia-cfa** [comment]
@embarnard I chose to merge `main` into this branch, which may cause you some local git troubles. The reason I did so was that I wanted to test out Heroku review apps. As a result, I found that Heroku review apps aren't working, which is a separate issue, not related to this pull request.

**Comment by alaroia-cfa** [comment]
@embarnard The changes you just pushed resolve the i18n-health issues. I'd say now feel free to merge!

**Pull request merged**